docs/index.rst

```diff
@@ -3,27 +3,43 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Liquid Prompt's documentation!
-=========================================
+Liquid Prompt's documentation
+=============================
 
-Liquid Prompt is an adaptive prompt for Bash & Zsh that gives you a nicely
+*Liquid Prompt* is an adaptive prompt for Bash & Zsh that gives you a nicely
 displayed prompt with useful information when you need it. It does this with a
 powerful theming engine and a large array of data sources.
 
-To get started, view the :doc:`install` documentation, which includes
+If you just want to have a short overview of the main Liquid Prompt features,
+see the ":doc:`overview`" section.
+
+To actually get started, view the :doc:`install` documentation, which includes
 instructions for trying Liquid Prompt temporarily.
 
+To know more about the available features, see the :doc:`theme` section,
+which have some explanations on what's displayed on your prompt,
+and the :doc:`config`, which lists all that change the behavior of the prompt.
```

Comment:
```suggestion
and the :doc:`config` section, which lists all that change the behavior of the prompt.
```

Reply:
I did not fixed this one, as the *compiled* text reads "and the Config Options", which seems enough, comparing to "and the Config Options section".